src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/UserTests.kt

```diff
@@ -51,16 +54,16 @@ class UserTests : RepositoryTests<UserRepository>()
     fun `can retrieve user with any case username`()
     {
         given(this::addTestUser).check { repo ->
-            assertThat(repo.getUserByUsername("test.user")).isNotNull()
-            assertThat(repo.getUserByUsername("Test.User")).isNotNull()
+            assertThat(repo.getUserByUsername("test.user", PermissionSet()) as User).isNotNull()
+            assertThat(repo.getUserByUsername("Test.User", PermissionSet()) as User).isNotNull()
```

Comment:
Ah good point. Casting just to check the right object is returned. Can just do without the assertions then, just calling the method and casting to User without an error being thrown will be a test.